### DISCUSSION TITLE
Update column to null programmatically

### DISCUSSION THREAD

**Discussion by wil2106** [supabase/supabase#15279] [Questions] [answered]
Hello,
I'm trying to update a column to NULL with Supabase Swift Client by setting the object property to nil, but the update ignores it.
I'm not sure if this is a Supabase Swift Client problem or a Supabase problem.
Is there a way to do it ?

**Comment by steve-chavez** [comment, marked as answer]
Updating to null is tested to work: https://github.com/PostgREST/postgrest/blob/078c6ec08c69da1482604e4688e2d19ec7c32a2f/test/spec/Feature/Query/UpdateSpec.hs#L122-L127

Pretty sure this works on `supabase-js` as well. It might be a bug on the swift client. I suggest raising an issue on its repo.
